src/sentry_json.c

```diff
@@ -367,6 +368,28 @@ sentry__jsonwriter_write_int32(sentry_jsonwriter_t *jw, int32_t val)
     }
 }
 
+void
+sentry__jsonwriter_write_int64(sentry_jsonwriter_t *jw, int64_t val)
+{
+    if (can_write_item(jw)) {
+        char buf[24];
+        snprintf(buf, sizeof(buf), "%" PRId64, val);
+        write_str(jw, buf);
+    }
+}
+
+void
+sentry__jsonwriter_write_uint64(sentry_jsonwriter_t *jw, uint64_t val)
+{
+    // TODO currently all uint64 values should be sent as strings; update when
+    // this changes
```

Comment:
what happened? what is this needed for? i was't part of the original reviews and might be missing some background context but i tried reverting "always send uint64 as string" and the unit tests pass. :)